src/python/espressomd/collision_detection.pyx

```diff
@@ -60,39 +60,42 @@ class CollisionDetection(ScriptInterfaceHelper):
 
         Parameters
         ----------
-        mode : One of "off", "bind_centers", "bind_at_point_of_collision", "bind_three_particles", "glue_to_surface"
-               Collision detection mode
+        mode : str, \{"off", "bind_centers", "bind_at_point_of_collision", "bind_three_particles", "glue_to_surface"\}
+            Collision detection mode
 
         distance : :obj:`float`
-               Distance below which a pair of particles is considered in the collision detection
+            Distance below which a pair of particles is considered in the
+            collision detection
 
-        bond_centers : Instance of :class:`espressomd.interactions.BondedInteraction`
-               Bond to add between the colliding particles
+        bond_centers : :obj:`espressomd.interactions.BondedInteraction`
+            Bond to add between the colliding particles
 
-        bond_vs :  Instance of :class:`espressomd.interactions.BondedInteraction`
-               Bond to add between virtual sites (for modes using virtual sites)
+        bond_vs : :obj:`espressomd.interactions.BondedInteraction`
+            Bond to add between virtual sites (for modes using virtual sites)
 
         part_type_vs : :obj:`int`
-               Particle type of the virtual sites being created on collision (virtual sites based modes)
+            Particle type of the virtual sites being created on collision
+            (virtual sites based modes)
 
         part_type_to_be_glued : :obj:`int`
-               particle type for "glue_to_surface" mode. See user guide.
+            particle type for ``"glue_to_surface"`` mode. See user guide.
 
         part_type_to_attach_vs_to : :obj:`int`
-               particle type for "glue_to_surface" mode. See user guide.
+            particle type for ``"glue_to_surface"`` mode. See user guide.
 
         part_type_after_glueing : :obj:`int`
-               particle type for "glue_to_surface" mode. See user guide.
+            particle type for ``"glue_to_surface"`` mode. See user guide.
 
         distance_glued_particle_to_vs : :obj:`float`
-               Distance for "glue_to_surface" mode. See user guide.
+            Distance for ``"glue_to_surface"`` mode. See user guide.
 
-        bond_three_particles : Instance of :class:`espressomd.interactions.BondedInteraction`
-               First angular bond for the "bind_three_particles" mode. See user guide
+        bond_three_particles : :obj:`espressomd.interactions.BondedInteraction`
+            First angular bond for the ``"bind_three_particles"`` mode. See
+            user guide
 
         three_particle_binding_angle_resolution : :obj:`int`
-              Resolution for the angular bonds (mode "bind_three_particles").
-              Resolution+1 bonds are needed to accommodate the case of a 180 degrees
+            Resolution for the angular bonds (mode ``"bind_three_particles"``).
+            Resolution+1 bonds are needed to accommodate the case of a 180 degrees
```

Comment:
[...] the case of ~a~ 180 degrees [...]
or add missing words ([...] of a 180 degrees turn?)

Reply:
based on [the user guide](http://espressomd.org/html/doc/advanced_methods.html#creating-bonds-when-particles-collide), point `"bind_three_particles"`, maybe they meant this:
```suggestion
            Resolution+1 bonds are needed to accommodate the case of 180 degrees
            angles
```